cmake/Modules/FindComputeCpp.cmake

```diff
@@ -375,15 +418,9 @@ function(__build_ir)
     # Add both source and the sycl files to the VS solution.
     target_sources(${SDK_BUILD_IR_TARGET} PUBLIC ${SDK_BUILD_IR_SOURCE} ${outputSyclFile})
 
-    # NOTE: The Visual Studio generators parse compile flags differently,
-    # hence the different argument syntax
-    if(CMAKE_GENERATOR MATCHES "Visual Studio")
-      set(forceIncludeFlags "/FI\"${includedFile}\" /TP")
-    else()
-      set(forceIncludeFlags /FI ${includedFile} /TP)
-    endif()
+    set(forceIncludeFlags "/FI ${includedFile} /TP")
   else()
-      set(forceIncludeFlags "-include ${includedFile} -x c++")
+    set(forceIncludeFlags "-include ${includedFile} -x c++")
```

Comment:
This is good, thanks for doing this

Reply:
I don't know why it did not work before. Might have been an MSBuild bug. It all works now.